js/package.json

```diff
@@ -53,53 +53,52 @@
     "npm-release.sh"
   ],
   "dependencies": {
-    "@types/flatbuffers": "1.9.0",
-    "@types/node": "10.5.1",
-    "@types/text-encoding-utf-8": "1.0.1",
-    "command-line-args": "5.0.2",
-    "command-line-usage": "5.0.5",
-    "flatbuffers": "1.9.0",
-    "json-bignum": "0.0.3",
-    "text-encoding-utf-8": "1.0.2",
-    "tslib": "1.9.3"
+    "@types/flatbuffers": "^1.9.0",
+    "@types/node": "^10.10.3",
+    "@types/text-encoding-utf-8": "^1.0.1",
+    "command-line-args": "^5.0.2",
+    "command-line-usage": "^5.0.5",
+    "flatbuffers": "^1.9.0",
+    "json-bignum": "^0.0.3",
+    "text-encoding-utf-8": "^1.0.2",
+    "tslib": "^1.9.3"
   },
   "devDependencies": {
     "@std/esm": "0.26.0",
-    "@types/glob": "5.0.35",
-    "@types/jest": "22.2.3",
-    "babel-jest": "22.4.3",
+    "@types/glob": "7.1.0",
+    "@types/jest": "23.3.2",
     "benchmark": "2.1.4",
     "coveralls": "3.0.2",
     "del": "3.0.0",
-    "glob": "7.1.2",
-    "google-closure-compiler": "20180610.0.2",
+    "glob": "7.1.3",
+    "google-closure-compiler": "20180910.0.0",
     "gulp": "github:gulpjs/gulp#6d71a658c61edb3090221579d8f97dbe086ba2ed",
     "gulp-json-transform": "0.4.5",
-    "gulp-rename": "1.3.0",
+    "gulp-rename": "1.4.0",
     "gulp-sourcemaps": "2.6.4",
-    "gulp-typescript": "4.0.2",
+    "gulp-typescript": "5.0.0-alpha.3",
     "ix": "2.3.5",
-    "jest": "22.4.3",
+    "jest": "23.6.0",
     "jest-environment-node-debug": "2.0.0",
     "json": "9.0.6",
-    "lerna": "2.11.0",
-    "lint-staged": "7.2.0",
+    "lerna": "3.4.0",
+    "lint-staged": "7.3.0",
     "merge2": "1.2.2",
     "mkdirp": "0.5.1",
     "npm-run-all": "4.1.3",
     "pump": "3.0.0",
     "rimraf": "2.6.2",
-    "rxjs": "5.5.6",
-    "shx": "0.3.1",
-    "source-map-loader": "0.2.3",
+    "rxjs": "5.5.11",
+    "shx": "0.3.2",
+    "source-map-loader": "0.2.4",
+    "terser-webpack-plugin": "1.1.0",
     "trash": "4.3.0",
     "ts-jest": "22.4.6",
-    "ts-node": "7.0.0",
-    "tslint": "5.10.0",
+    "ts-node": "7.0.1",
+    "tslint": "5.11.0",
     "typedoc": "github:typestrong/typedoc#1780beb1448d",
-    "typescript": "2.9.2",
-    "uglifyjs-webpack-plugin": "1.1.6",
-    "webpack": "4.14.0",
+    "typescript": "3.0.3",
```

Comment:
Would you mind bumping this to TS 3.1, and typedoc 0.13 (which supports 3.1)? Keeping the TS and typedoc versions in sync saves `npm install` from downloading two copies of typescript.

Reply:
Done